test-lib.sh

```diff
@@ -221,7 +255,8 @@ function ct_npm_works() {
     return 1
   fi
 
-  docker run --rm ${IMAGE_NAME} /bin/bash -c "npm install jquery && test -f node_modules/jquery/src/jquery.js"
+  docker run $(ct_mount_ca_file) --rm ${IMAGE_NAME} /bin/bash -c "npm install jquery && test -f node_modules/jquery/src/jquery.js"
```

Comment:
Missing quotes

Reply:
It has to be w/o quotes, otherwise you see error like
```bash
/usr/bin/docker-current: Error response from daemon: create  /etc/pki/ca-trust/source/anchors/RH-IT-Root-CA.crt: " /etc/pki/ca-trust/source/anchors/RH-IT-Root-CA.crt" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intented to pass a host directory, use absolute path.
```

Reply:
Any reason to have the space before the file path?

Reply:
Oh nvm, I see the issue now. Yeah it would not really work quoted since it is an argument to the docker applicaton, my bad.